src/openvic-simulation/scripts/Condition.hpp

```diff
@@ -63,12 +63,13 @@ namespace OpenVic {
 		BUILDING        = 1 << 20,
 		CASUS_BELLI     = 1 << 21,
 		GOVERNMENT_TYPE = 1 << 22,
-		MODIFIER        = 1 << 23,
+		COUNTRY_EVENT_MODIFIER = 1 << 23,
 		NATIONAL_VALUE  = 1 << 24,
 		CULTURE_UNION   = 1 << 25, // same as COUNTRY_TAG but also accepts scope this_union
 		CONTINENT       = 1 << 26,
 		CRIME           = 1 << 27,
 		TERRAIN         = 1 << 28,
+		PROVINCE_EVENT_MODIFIER = 1 << 29
```

Comment:
Move this to just below `COUNTRY_EVENT_MODIFIER` and change the bitshift numbers to follow the new order. Also move `BUILD_STRING(PROVINCE_EVENT_MODIFIER)` to below `BUILD_STRING(COUNTRY_EVENT_MODIFIER)` in the print operator function below.

Reply:
I'm used to appending bit flag enums for backwards compatiblity but you're right. In this case we don't need that.